{"name": "Put method check in case of a correct payload", "status": "broken", "statusDetails": {"message": "TypeError: 'headers' is an invalid keyword argument for print()", "trace": "@allure.title(\"Put method check in case of a correct payload\")\n    def test_put_200():\n        try:\n            random_payload = {\n            \"timestamp\": int(time.time()),\n            \"number\": random.randint(1, 100)\n            }\n            updated_payload = {\n            \"timestamp\": int(time.time()),\n            \"number\": random.randint(1, 100)*100\n            }\n    \n            post_response_id = request.post(json.dumps(random_payload)).json()[\"_id\"]\n            get_response_initial_size = len(request.get().json())\n>           put_response = request.put(post_response_id,updated_payload)\n\ntests/test_cases.py:104: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <python_rest_api.pytest_based_framework.requests.https_request object at 0x7fa7951d72e0>, id = '62a1dbfd6f047803e8ae84f5'\npayload = {'number': 6300, 'timestamp': 1654774779}\n\n    def put(self, id, payload):\n        print('id-------', self.url+\"/\"+str(id))\n>       print('header---------', headers=self.headers)\nE       TypeError: 'headers' is an invalid keyword argument for print()\n\npytest_based_framework/requests.py:22: TypeError"}, "start": 1654774779885, "stop": 1654774781779, "uuid": "ff7a3af6-01b1-4761-bd02-3615781e1fb4", "historyId": "87fe87f081e5c1389207eeed8a6b59f1", "testCaseId": "b4b2a9526a3e94e429b66d5c0c490efc", "fullName": "tests.test_cases#test_put_200", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_cases"}, {"name": "host", "value": "KSargsyanL2M"}, {"name": "thread", "value": "62337-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_cases"}]}